sccp/figures/figure3.py

```diff
@@ -57,7 +57,7 @@ def makeFigure():
     plotDrugUMAP(drugs, "Pf2", dataDF["Drug"].values, pf2Points, ax[8:10])
     plotDrugUMAP(drugs, "PCA", dataDF["Drug"].values, pcaPoints, ax[10:12])
 
-    cmp = ["Cmp. 23", "Cmp. 25", "Cmp. 29", "Cmp. 30"]
+    cmp = ["Cmp. 24", "Cmp. 25", "Cmp. 12", "Cmp. 15"]
```

Comment:
Maybe we should just remove these, since the component-by-component cell weights are all plotted in Fig. 3a?